src/components/Head/Head.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import { useState, useEffect } from 'react';
+import { Helmet } from 'react-helmet';
+import { withRouter } from 'react-router';
+import WithCurrentUser from '../HOCs/WithCurrentUser/WithCurrentUser';
+import WithCurrentProject from '../AdminPane/HOCs/WithCurrentProject/WithCurrentProject';
+import WithCurrentChallenge from '../AdminPane/HOCs/WithCurrentChallenge/WithCurrentChallenge';
+import { injectIntl } from 'react-intl';
+import _get from 'lodash/get'
+import _isEmpty from 'lodash/isEmpty'
+
+const HeadTitle = (props) => {
+  
+  const REACT_APP_TITLE = 'MapRoulette'
```

Comment:
put this variable outside of the component.  as much as possible, make your components lightweight.  no need to redeclare a variable or a pure function.